charts/etcd/templates/etcd-statefulset.yaml

```
@@ -213,6 +213,14 @@ spec:
 {{- if .Values.etcd.heartbeatDuration }}
         - --enable-member-lease-renewal=true
         - --k8s-heartbeat-duration={{ .Values.etcd.heartbeatDuration }}
+{{- end }}
+{{- if .Values.backup.leaderElection }}
+        {{- if .Values.backup.leaderElection.reelectionPeriod }}
+        - --etcd-connection-timeout-leader-election={{ .Values.backup.leaderElection.reelectionPeriod }}
+        {{- end }}
+        {{- if .Values.backup.leaderElection.etcdConnectionTimeout }}        
+        - --reelection-period={{ .Values.backup.leaderElection.etcdConnectionTimeout }}
+        {{- end }}
```

```suggestion
        {{- if .Values.backup.leaderElection.reelectionPeriod }}
        - --etcd-connection-timeout-leader-election={{ .Values.backup.leaderElection.etcdConnectionTimeout }}
        {{- end }}
        {{- if .Values.backup.leaderElection.etcdConnectionTimeout }}        
        - --reelection-period={{ .Values.backup.leaderElection.reelectionPeriod }}
        {{- end }}
```
These two values are interchanged

---

thanks for catching this.